tests/integration/test_main.py

```diff
@@ -510,6 +514,65 @@ def test_successful_parse(self, tmp_path):
         for _, count in record_count_by_type.items():
             assert count > 0
 
+    def test_successful_parse_of_aggregated_capture_file(self, tmp_path):
```

Comment:
I would like a test that checks that the **aggregated** data at the high watermark using both reporters is the same for some bunch of code examples.